sys-devel/llvm/llvm-16.0.0_rc2.ebuild

```
@@ -325,6 +325,10 @@ get_distribution_components() {
 }
 
 multilib_src_configure() {
+	#GCC -flto ICE protection #873670
+	if tc-is-gcc; then
+		filter-lto
+	fi
```

For such a short construct, it's fine to oneline it into:

```suggestion
	tc-is-gcc && filter-lto
```